pkg/constants/constants.go

```diff
@@ -231,6 +231,7 @@ const (
 
 var (
 	LocalGatewayHost = "knative-local-gateway.istio-system.svc." + network.GetClusterDomainName()
+	IstioMeshGateway = "mesh"
```

Comment:
is it safe to hardcode to mesh?
I mean, it is isolated peer namespace, but the MeshConfig can have m ore than one namespace, I don't think it is the case for KServe, just double checking :)

Reply:
It should be safe.

The _mesh_ gateway is a keyword in the VirtualService. It means that the configuration should be applied to the sidecars.

Reply:
One little nit - it is applied to ALL sidecars in the mesh.